### ISSUE TITLE
Characters with accents display funny

### ISSUE THREAD

**Issue by raarce** [arturadib/strapdown#26]
Characters with accents display funny. Any workarounds? Thanks

Example: `multiplicaciÃ³n` instead of `multiplicación`. Example document: [http://knuth.uprrp.edu/~rarce/tgh.html](http://knuth.uprrp.edu/~rarce/tgh.html)

**Comment by raarce** [comment]
Awesome product by the way!

**Comment by memeplex** [comment]
Have you specified your charset?

```
<meta http-equiv="content-type" content="text/html; charset=utf-8">
```

This is not markdown, strapdown or marked related.

I can see you hablas español by the way.

**Comment by ccom3033** [comment]
Issue solved. Html:1, Me:0. Gracias!

**Comment by memeplex** [comment]
Hopefully you will be interested in this strapdown template I've shared as a gist:

https://gist.github.com/memeplex/6309540

Besides adding utf8 support it fixes some issues and allows for some additional goodies.

**Comment by walterdavis** [comment]
This is a nice start. You may want to move the last closing comment down a couple of lines -- you missed the closing </script> tag...

Walter

**Comment by memeplex** [comment]
You're right. I did some last minute editions with the risks which that always implies.

**Comment by walterdavis** [comment]
Is there any documentation for the themes supported by strapdown? I fiddled around with these: http://bootswatch.com/2/ and some of them worked, while others did not.

Walter

**Comment by memeplex** [comment]
If you give a look at the strapdown sources you'll notice the themes are being downloaded relatively to the strapdown.js location. So I guess you can use only the provided themes if you don't want to download and edit strapdown.js yourself.
